packer/bsd/openbsd_vanilla.pkr.hcl

```diff
@@ -42,6 +46,9 @@ source "qemu" "qemu-gce-builder" {
   format                  = "raw"
   vm_name                 = "disk.raw"
   output_directory        = "output"
+  qemuargs                = [
+    [ "-serial", "stdio" ]
```

Comment:
Looks like that didn't work, neither net/openbsd shows more details. I suspect the reason we don't see more is because you didn't export PACKER_LOG:
https://github.com/anarazel/pg-vm-images/blob/main/.cirrus.yml#L168
that way it won't be visible to packer afaics.

Reply:
Yes, my bad. It works now.

Reply:
I think we should split the serial console stuff into its own PR. The rest here we can apply now, but since the serial stuff isn't yet working properly...